### ISSUE TITLE
'(x-1)(x-2)' example: if we assume/accept that this is a function call, then it should give an error rather than -1

### ISSUE THREAD

**Issue by davidedc** [davidedc/Algebrite#69]
see discussion in https://github.com/davidedc/Algebrite/issues/68

**Comment by pabloahb** [comment]
Related:	

In Algebrite
		
Input: factor(x^2 - 3 x + 2)		
Output: (x - 2) (x - 1)     //With a space between the parentheses

Input: (x - 2) (x - 1)
Output: -2

Input: (x - 2)*(x - 1)
Output: x^2 - 3 x + 2

") (" and ")(" are giving a wrong output
") (" is used in the output of factor()

**Comment by davidedc** [comment]
note to self - RE:@pabloahb 's comment: see print.coffee line 1455

**Comment by pabloahb** [comment]
In 2017 I made this change and it worked:

https://github.com/pabloahb/Algebrite/commit/5cc82fa86b21391f6b2ef6460ebac08de8ab8829

```
sources/print.coffee
@@ -1447,12 +1447,12 @@ print_multiply_sign = ->
 	accumulator = ""
 	if printMode == PRINTMODE_LATEX
 		if printMode == PRINTMODE_PLAIN and !test_flag
-			accumulator += print_str(" ")
+			accumulator += print_str("*")
 		else
 			return accumulator
 
 	if printMode == PRINTMODE_PLAIN and !test_flag and !codeGen
-		accumulator += print_str(" ")
+		accumulator += print_str("*")
 	else
 		accumulator += print_str("*")
 	return accumulator

```

I did'n delete the last "if" just in case I have to change the symbol again

**Comment by davidedc** [comment]
@pabloahb indeed I think I'll do something like that

**Comment by davidedc** [comment]
OK this is now addressed. Note that it's not really possible to throw an error in cases such as
```
(x-2)(x-1)
```
because I don't want to exclude that ```(x-2)``` might be a function.

After all, ```(f-g)``` might be a function. The "arithmetics of functions" is not implemented yet, but it's unclear to me why/how to rule-out that ```(x-2)``` is a function.

However, at least it doesn't return ```-2``` or other nonsense of that kind.

Also, the printouts now by default add the "*" between the parentheses, and I fixed the example in the home page.

So, what could be conceivably done has been done, let me know if you have further thoughts.
